### PR TITLE
[DOCS-6931] Mobile Workspace 1.5

### PR DESCRIPTION
**Pull request by oliveralfresco2018**

### CODE REVIEW

File: mobile-workspace/latest/using/index.md [hunk 1]
```diff
@@ -154,32 +153,82 @@ Filters can be set for Status, Due date, and Task Name. Filter chips are shown a
...
 
 4. The filter chips will show the active filters and the refresh button will reset the filters.
 
 ### Task view
 
-Within the Task view are details about the task: Task name, Due date, Priority, Assignee, Status, and Identifier. There is a back arrow to exit Task view and return to the Task list.
+Within the Task view are details about the task: Task name, Due date, Priority, Assignee, Status, and Identifier. There is a back arrow to exit the Task view and return to the Task list.
 
-Individual tasks have a “Comments” section below the task details. The most recent comment is shown with the user's icon, user name, date the comment was added, and the user’s comment. If there is more than one comment, you will see a link to **View all**. The link **Add a comment…** is below existing comments.
+Individual tasks have a Comments section below the task details. The most recent comment is shown with the user's icon, user name, date the comment was added, and the user’s comment. If there is more than one comment, you will see a link to **View all**. The link **Add a comment…** is below the existing comments. If you have access to
```
Comment (lilustig) [inline mobile-workspace/latest/using/index.md:168]: not sure what is ' if you have access to ...' no full stop or did you intend to continue this ?

File: mobile-workspace/latest/using/index.md [hunk 2]
```diff
@@ -154,32 +153,82 @@ Filters can be set for Status, Due date, and Task Name. Filter chips are shown a
...
+
+You can reassign a task to another person.
+
+1. Open a task from the Task list.
+
+2. Press **Edit** on the top right.
+
+3. Press the **Edit** icon next to the **Assignee** name.
+
+4. Search for a new assignee using their name or email address.
```
Comment (lilustig) [inline mobile-workspace/latest/using/index.md:200]: you need to enter their full email address

**Review by lilustig — CHANGES_REQUESTED** [review]

File: mobile-workspace/latest/using/index.md (hunk 1 above)
Comment (oliveralfresco2018) [reply mobile-workspace/latest/using/index.md:168]: Deleted, thanks.

File: mobile-workspace/latest/using/index.md (hunk 2 above)
Comment (oliveralfresco2018) [reply mobile-workspace/latest/using/index.md:200]: I dont think any changes need to be made here, I have been through the process again now. You can search by name and email address. I dont need to mention full email address as that doesnt make any sense to me :-)

File: mobile-workspace/latest/using/index.md [hunk 3]
```diff
@@ -154,32 +153,82 @@ Filters can be set for Status, Due date, and Task Name. Filter chips are shown a
...
+
+1. Open a task from the Task list.
+
+2. Press **Add attachments**, at the bottom of the window.
+
+    You can add attachments by either taking a photo or video using your mobile device, by uploading a photo or video using storage accessible to your mobile device, or you can upload any other types of files that are accessible to your mobile device. Once you have added a task they will display below the **Attached files** section at the bottom of the window.
+
+    > **Note:** You may need to scroll further down the page to see all the attachments.  
+
+![add attachment]({% link mobile-workspace/images/add-attachment.png %})
```
Comment (anxumalo) [inline mobile-workspace/latest/using/index.md:234]:
```suggestion
![Add attachment]({% link mobile-workspace/images/add-attachment.png %})
```

**Review by anxumalo — APPROVED** [review]
LGTM

* `mobile-workspace/images/add-attachmenttest.png` isn't used in any file - please remove it

**Review by anxumalo — APPROVED** [review]
LGTM

**Pull request merged**